### DISCUSSION TITLE
Добро пожаловать в «Обсуждения» mnemonic-kb-ru!!

### DISCUSSION THREAD

**Discussion by dotland** [dotland/mnemonic-kb-ru#1] [Announcements]
## 👋 Добро пожаловать!
   Мы используем «Обсуждения» как место для общения с другими членами нашего сообщества. Мы надеемся, что вы будете:
   * задавать вопросы, которые вас интересуют,
   * поделиться идеями,
   * взаимодействовать с другими членами сообщества.

Помните, что это сообщество, которое мы строим вместе 💪.

**Comment by antonyveyre** [comment]
Спасибо, добрый человек. На работе мнемо-раскладка винды, дома – мак, нужно было привести к общему знаменателю. И вот теперь спокойно набираю этот текст с Вашей помощью. теперь смогу проще печатать по-русски, меньше нагружать мозг.
